src/labels/CustomLabels.labels

```diff
@@ -6718,7 +6790,303 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
+        <value>This brief overview is meant to dip your toes into what’s possible with NPSP, but you can dive deeper and discover the many other features available to your organization.</value>
+    </labels>
+    <labels>
+        <fullName>gsChecklistItemNPSPCloudPriBtnLabel</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Button label of checklist item NPSP Cloud in Get Started</shortDescription>
+        <value>See Features</value>
+    </labels>
+     <labels>
+        <fullName>gsChecklistItemNPSPCloudLinkLabel</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Link label of checklist item NPSP Cloud in Get Started</shortDescription>
+        <value>Learn more about NPSP Features in Trailhead</value>
+    </labels>
+    <labels>
+        <fullName>gsChecklistItemNPSPCloudTitle</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Title of checklist item NPSP Cloud in Get Started</shortDescription>
+        <value>Learn about other Nonprofit Cloud Features</value>
+    </labels>
+    <labels>
+        <fullName>gsChecklistItemNPSPGuideDesc</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Description of checklist item NPSP Guide in Get Started</shortDescription>
+        <value>Take the time to carefully plan your implementation and ensure your Salesforce org meets the needs of your nonprofit.</value>
+    </labels>
+    <labels>
+        <fullName>gsChecklistItemNPSPGuidePriBtnLabel</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Button label of checklist item NPSP Guide in Get Started</shortDescription>
+        <value>Download PDF</value>
+    </labels>
+    <labels>
+        <fullName>gsChecklistItemNPSPGuideTitle</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Title of checklist item NPSP Guide in Get Started</shortDescription>
+        <value>NPSP Planning Guide</value>
+    </labels>
+    <labels>
+        <fullName>gsChecklistItemNPSPSettingsDesc</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Description of checklist item NPSP Settings in Get Started</shortDescription>
+        <value>Navigate to the NPSP Settings page and do a thorough review to confirm the settings are correct for your organization and functioning the way you expect them to.</value>
```

Comment:
Can you please add the word "are" before functioning so that this reads: Navigate to the NPSP Settings page and do a thorough review to confirm the settings are correct for your organization and are functioning the way you expect them to.